inputremapper/configs/input_config.py

```diff
@@ -55,7 +55,7 @@ class InputConfig(BaseModel):
     # origin_hash is a hash to identify a specific /dev/input/eventXX device.
     # This solves a number of bugs when multiple devices have overlapping capabilities.
     # see utils.get_device_hash for the exact hashing function
-    origin_hash: Optional[DeviceHash] = None  # type: ignore
+    origin_hash: Optional[DeviceHash] = None
```

Comment:
I would like to ensure the origin_hash is always lowercase. It is a big hex number, so it should be case insensitive.

Reply:
Okay I see just now, you added a valitator for that.